src/views/NewRideView.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
```

Comment:
Super, sieht richtig gut aus!
Gabriel (und Joe) nehmen oft noch das "form"-tag zum wrappen der input-Element, das hat wohl irgendwelche Vorteile (die ich tatsächlich auch nicht weiß gerade)?

Reply:
Habs mal gemacht, können wir vielleicht auch nochmal nachfragen, was die Vorteile davon sind. Und muss "action" anscheinend in "method" ändern. Mach ich gleich und pushe es nochmal